src/components/LanguageSelector.tsx

```diff
@@ -0,0 +1,112 @@
+import { SelectItem } from '@/types';
```

Comment:
FYI: The language selection menu was implemented separately, so this file was removed in #40.  🙏  

https://github.com/miurla/babyagi-ui/pull/40